app/models.py

```diff
@@ -99,103 +87,119 @@ def __repr__(self):
 class IngredientNutrient(db.Model):
     __tablename__ = "ingredient_nutrient"
 
-    ingredient_id     = db.Column(db.Integer, db.ForeignKey("ingredient.ingredient_id"), primary_key=True)
+    ingredient_id     = db.Column(db.Integer,\
```

Comment:
Just a really minor detail but python doesn't require new lines to be escaped inside of parenthesizes expressions. I think that applies to bracketed and curly-brace expressions too such as in list and dictionary definitions. But again it's very minor and it comes down to choice either way, so we can merge this now and remove the escapes in later phases.